gateleen-cache/src/main/java/org/swisspush/gateleen/cache/storage/CacheStorage.java

```diff
@@ -1,6 +1,7 @@
 package org.swisspush.gateleen.cache.storage;
 
 import io.vertx.core.Future;
+import io.vertx.core.Promise;
```

Comment:
Import looks unused as its the only change in the file

Reply:
Yep, thx, will fix